services/port_manager/src/test/java/com/futurewei/alcor/portmanager/config/UnitTestConfig.java

```diff
@@ -82,4 +85,42 @@
             "        \"allowed_address_pairs\":[{\"ip_address\":\"" + ip2 + "\", \"mac_address\":\"" + mac1 + "\"}]\n" +
             "    }\n" +
             "}";
+    public static String createPortBulk = "{\n" +
```

Comment:
Bulk APIs are very fancy. Let us add some UTs to cover them :-)

Reply:
There are two: createPortBulkTest and updateMacAddressAndFixedIpsBulkTest

Reply:
Oh I see. I didn't find it in this PR. Are they checked in previously?

Reply:
Never mind. Find it now.